ui-core/src/main/java/com/adyen/checkout/components/ui/view/address/AddressSpecification.kt

```diff
@@ -3,14 +3,13 @@
  *
  * This file is open source and available under the MIT license. See the LICENSE file for more info.
  *
- * Created by ozgur on 31/10/2022.
- *
+ * Created by onurk on 13/1/2023.
  */
 
-package com.adyen.checkout.card.ui
```

Comment:
This class should be in `com.adyen.checkout.components.ui` as well I think

Reply:
It is under the com.adyen.checkout.components.ui.view.address. Do you mean I  exactly should put it under com.adyen.checkout.components.ui?

Reply:
Yes, that's what I mean